### PR TITLE
Plane actions

### PR DESCRIPTION
**Pull request by aggie-f**
Hey, Finished the actions and views for planes. Please check

### CODE REVIEW

**Comment by eadysr** [comment]
Looks good to me :)

**Comment by eadysr** [comment]
Agata and I worked on it together so if one of you guys wants to merge that'd be great :)

File: app/controllers/planes_controller.rb [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
   def index
+    @planes = Plane.all
   end
 
   def show
+    @plane = Plane.find(params[:id])
   end
 
   def new
+    @owner = current_user.id
```
Comment (mroses33) [inline app/controllers/planes_controller.rb]: should this also be @owner = current_user

Comment (aggie-f) [reply app/controllers/planes_controller.rb]: Thanks, changing now

**Pull request merged**